Big-Spender/readme.md

```diff
@@ -69,22 +74,28 @@ INSERT YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERY HERE
-```
+select * from spends where description ilike '%fee%';
+```  
 
-**Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
+**Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense "expense_areas" area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
 INSERT YOUR QUERY HERE
+  select * from spends join expense_areas on expense_areas.id = spends.expense_area_id 
+big_spender-# where expense_areas.id = 2;   
```

Comment:
Can we clean this up a bit so we don't have the database name here? 
how did you get the expense_areas? How would someone else know what 2 is?

Reply:
select * from spends join expense_areas on expense_areas.id = spends.expense_area_id where expense_area ilike '%Better Hospital Food%';